Example/Example/Common/BankCard/BankCardViewController.swift

```diff
@@ -14,6 +14,50 @@
 
 import TextInputKit
 
+struct BankCard {
+
+    var cardNumber: BankCardNumber? = nil
+
+    var expiryDate: BankCardExpiryDate? = nil
+
+    var cardHolderName: String = ""
+
+    var securityCode: String = ""
+
+}
+
+extension BankCard {
+
+    var prettyDescription: String {
+        let cardNumberDescription: String = {
```

Comment:
As I can see, in the current case it won't improve the code readability. But it could be useful in a more general case, yes.